### PR TITLE
Feature/DEV-22 languages page

### PR DESCRIPTION
**Pull request by AAKlimkov**
add languages page

### CODE REVIEW

**Review by shev7 — CHANGES_REQUESTED** [review]
same issues as with previous PRs

**Comment by AAKlimkov** [comment]
1. Added export of languageList
2. Remove unused fragment in languageList
3. pass initialValues to the form in languageDialog
4. Remove unused interface in languageList
5. move columnConfigs from languageList component

File: src/modules/CVs/components/details/details.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import CvDetailsForm from "@/modules/CVs/components/cv-details-form/cv-details-form.tsx";
+import CvDetailsForm from "@/modules/cvs/components/cv-details-form/cv-details-form.tsx";
```
Comment (shev7) [inline src/modules/CVs/components/details/details.tsx]:
```suggestion
import CvDetailsForm from "@/modules/cvs/components/cv-details-form/cv-details-form";
```

File: src/modules/languages/shemas/language.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { z } from "zod";
+import i18n from "@/i18n";
+
+export const getLanguageFormSchema = () => {
+    const iso2Message = { message: i18n.t("fieldErrors.nameRequired") };
```
Comment (shev7) [inline src/modules/languages/shemas/language.ts]: why key is `nameRequired` if its iso2Message

File: src/modules/languages/ui/languages-dialog.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+interface FormData {
+    language: string;
+    native_name: string;
+    iso2: string;
+}
+interface LanguageDialogProps {
+    title: string;
+    onConfirm: (formData: FormData) => void;
+    onClose: () => void;
+    initialValues: { language: string; native_name: string; iso2: string };
```
Comment (shev7) [inline src/modules/languages/ui/languages-dialog.tsx]: this type is the same as `FormData` interface above

```suggestion
    initialValues: FormData;
```

File: src/modules/languages/ui/languages-list.tsx [hunk 4]
```diff
@@ -0,0 +1,88 @@
+import { useState } from "react";
+import { Language } from "cv-graphql";
+import { t } from "i18next";
+import ListTemplate from "@/components/list-lemplate/list-template";
+import { Action } from "@/components/list-lemplate/actions-menu";
+import { ColumnConfig } from "@/components/list-lemplate/table-template";
+import { useConfirm } from "@/components/confirm-dialog/";
+import { useLanguageCreate, useLanguageDelete, useLanguagesQuery, useLanguageUpdate } from "../api";
+import { useAddLanguage } from "./languages-dialog";
```
Comment (shev7) [inline src/modules/languages/ui/languages-list.tsx]: if you are using this dialog to both create and update it should be named as useLanguageDialog or something like that, not addLanguage

**Review by shev7 — APPROVED** [review]

**Review by savelievamareena — APPROVED** [review]

**Review by RC-3222 — APPROVED** [review]

**Pull request merged**